### DISCUSSION TITLE
Is it possible to access the "Content-Disposition" header from $fetch response?

### DISCUSSION THREAD

**Discussion by jshimkoski** [nuxt/nuxt#16891] [Questions]
Hello all,

Is it possible to access the "Content-Disposition" header from $fetch response?

If so, does anyone have an example of how to accomplish this?

Thank you!

**Comment by danielroe** [comment]
You can use `$fetch.raw`.

**Reply by jshimkoski** [reply]
Do headers only come back with `$fetch.raw` on the server-side? All I see is any empty object on the client-side.
